imap/jmap_contact.c

```diff
@@ -1715,11 +1708,19 @@ static json_t *jmap_contact_from_vcard(const char *userid,
                                 jmap_utf8string(strarray_safenth(a, 5)));
             json_object_set_new(item, "country",
                                 jmap_utf8string(strarray_safenth(a, 6)));
-            if (countrycode) {
-                buf_setcstr(&buf, countrycode);
-                buf_lcase(&buf);
-                json_object_set_new(item, "countryCode",
-                        jmap_utf8string(buf_cstring(&buf)));
+
+            /* Read countryCode from same-grouped ABADR property, if any */
+            const char *countrycode = NULL;
+            if (entry->group) {
+                buf_setcstr(&buf, entry->group);
+                countrycode = hash_lookup(buf_ucase(&buf), &abadr_by_group);
```

Comment:
The line `const char *countrycode = NULL;` can be removed, and the line `countrycode = hash_lookup(buf_ucase(&buf), &abadr_by_group);` replaced with `const char *countrycode = hash_lookup(buf_ucase(&buf), &abadr_by_group);`.